diff --git a/Marksman/Marksman.fsproj b/Marksman/Marksman.fsproj
index 670f99c..773dd86 100644
--- a/Marksman/Marksman.fsproj
+++ b/Marksman/Marksman.fsproj
@@ -4,18 +4,8 @@
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
         <AssemblyName>marksman</AssemblyName>
-        <VersionPrefix>1.0.0</VersionPrefix>
         <EnableLint>false</EnableLint>
     </PropertyGroup>
-    <Target Name="Version" BeforeTargets="BeforeBuild">
-        <Exec Command="git describe --always --dirty" ConsoleToMSBuild="true">
-            <Output TaskParameter="ConsoleOutput" PropertyName="VersionString"/>
-        </Exec>
-        <PropertyGroup>
-            <VersionSuffix>$(VersionString)</VersionSuffix>
-            <Version>$(VersionPrefix)-$(VersionSuffix)</Version>
-        </PropertyGroup>
-    </Target>
     <Target Name="FSharpLint" AfterTargets="BeforeBuild" Condition="$(EnableLint)">
         <PropertyGroup>
             <LintCommand>dotnet fsharplint -f msbuild lint --lint-config $(MSBuildProjectDirectory)/../fsharplint.json $(MSBuildProjectFullPath)</LintCommand>
